packages/indexer/config.yaml

```diff
@@ -5,6 +5,10 @@ contracts:
     handler: src/EventHandlers.ts
     events:
       - event: ConditionModuleWhitelisted(address indexed conditionModule, string name, bool indexed whitelisted, address sender, uint256 timestamp)
+        requiredEntities:
+          - name: ConditionModule
+            labels: 
+              - conditionModule
```

Comment:
https://docs.envio.dev/docs/labels - an advanced feature :) (not required, but it can make the code cleaner is some cases, especially when you want to load arrays or sets of entities)


Aside:
Also, this schema is likely going to change - but we'll make it backwards compatible of course. Probably to:
```
entities:
  - ConditionModule:
    labels:
      - conditionModule
```